alembic/versions/ad13850a7245_ui_flip_project_icon.py

```diff
@@ -0,0 +1,43 @@
+"""Ui flip & project icon
+
+Revision ID: ad13850a7245
+Revises: 96fbb404381e
+Create Date: 2025-06-23 11:35:12.693674
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "ad13850a7245"
+down_revision = "96fbb404381e"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    connection = op.get_bind()
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column(
+        "user", sa.Column("use_new_cognition_ui", sa.Boolean(), nullable=True)
+    )
+    op.add_column(
+        "project", sa.Column("icon", sa.String(), nullable=True), schema="cognition"
+    )
+    # ### end Alembic commands ###
+
+    update_dataset_sql = """
```

Comment:
is this the update for the old projects? (no updater needed for the next release?)

Reply:
yes, no updater needed :)